src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/controllers/CalculateInterestAmountTest.java

```diff
@@ -0,0 +1,72 @@
+package uk.gov.hmcts.cmc.claimstore.controllers;
+
+import org.junit.Test;
+import org.springframework.http.HttpHeaders;
+import org.springframework.test.web.servlet.MvcResult;
+import org.springframework.test.web.servlet.ResultActions;
+import uk.gov.hmcts.cmc.claimstore.BaseSaveTest;
+import uk.gov.hmcts.cmc.domain.models.InterestAmount;
+
+import java.math.BigDecimal;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+public class CalculateInterestAmountTest extends BaseSaveTest {
+
+    @Test
+    public void shouldCalculatedAmount() throws Exception {
+        MvcResult result = makeRequest("2010-10-10", "2010-10-16", 30, 8000)
+            .andExpect(status().isOk())
+            .andReturn();
+
+        InterestAmount obj = deserializeObjectFrom(result, InterestAmount.class);
+        assertThat(obj.getAmount()).isEqualTo(BigDecimal.valueOf(39.45));
+    }
+
+    @Test
+    public void shouldReturnBadRequestWhenDateFromIsEmpty() throws Exception {
+        makeRequest("", "2010-10-10", 30, 8000).andExpect(status()
```

Comment:
I would love to break like before `.andExpect` here and in other places.

Reply:
done